sql/1.0xx/activity_feed.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN;
+
+CREATE TABLE "activity_feed" (
+  "id" serial NOT NULL,
+  "created" timestamp with time zone NOT NULL,
+  "type" text NOT NULL,
+  "description" text NOT NULL,
+  "format" text DEFAULT 'markdown' NOT NULL,
+  "for_user" bigint,
```

Comment:
so if we send a msg to every user and we have 300 users, we'll have to add 300 rows to this table.  is that ok?  depends how many users we get i guess.  and how many notifications.  and if we ever delete stuff from this table.